Classes/Extensions/DeclarativeProtocol+Apply.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Created by Kevin Guo on 2022/1/3.
+//
+
+import Foundation
+extension DeclarativeProtocol {
+
```

Comment:
Could you please remove empty line to make it same style with another extensions